crates/uv-distribution-filename/src/wheel.rs

```diff
@@ -104,6 +106,25 @@ impl WheelFilename {
         )
     }
 
+    /// Returns a consistent cache key with a maximum length of 64 characters.
+    ///
+    /// Prefers `{version}-{tags}` if such an identifier fits within the maximum allowed length;
+    /// otherwise, uses a truncated version of the version and a digest of the tags.
+    pub fn cache_key(&self) -> String {
+        const CACHE_KEY_MAX_LEN: usize = 64;
```

Comment:
@nkitsaini -- I changed this to 64 to accommodate common manylinux tags, like `cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64`. I don't see much of a downside. What do you think?